### ISSUE TITLE
Upgrade OpenSSL for Bionic

### ISSUE THREAD

**Issue by adityapatadia** [openresty/docker-openresty#151]
OpenSSL version is still 1.1.0 for bionic. https://github.com/openresty/docker-openresty/blob/master/bionic/Dockerfile#L16

TLS 1.3 requires OpenSSL 1.1.1.

**Comment by neomantra** [comment]
I see this in the `1.17.8.1` change log:

```
feature: added support for OpenSSL 1.1.1 by upgrading the OpenSSL patches.
Thanks spacewander for the patch.
```

So it is now supported by upstream, so I will update the `focal` and `bionic` packages.

See also #101

**Comment by neomantra** [comment]
This is released in `1.17.8.2-1`.
